tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
-    "lib": ["es2017"],
+    "lib": ["es2019"],
```

Comment:
Switched this to allow use of `Array.prototype.flat` which appears requires node 12 (https://node.green/#ES2019-features-Array-prototype--flat--flatMap-)

Reply:
Nice one, and as soon as we merge #1704 the build error should go way.